### PR TITLE
Fix delete private item from page and alert editing private item in page

### PR DESCRIPTION
**Pull request by leandrohmanica**
https://trello.com/c/uU3KzsK9/129-removing-private-item-from-page-ends-up-removing-every-instance-of-item-for-that-page

### CODE REVIEW

**Comment by longaraymatheus** [comment]
@lucaslongaray please help reviewing/testing this PR if you have bandwidth

File: config/locales/pt.yml [hunk 1]
```diff
@@ -426,6 +426,7 @@ pt:
     not_linked: -- sem vínculo --
     og_description: Aplicativo de Comunicação Alternativa que permite pessoas com dificuldades de fala e mobilidade se comunicarem utilizando pictogramas que representam palavras.
     page_destroy_alert: Apagar esta página também apagará todos seus pictogramas.
+    page_item_edit_alert: Este é um item privado, editando irá alterar todas as referências.
```
[the remark this answers is not in the document]
Comment (longaraymatheus) [reply config/locales/pt.yml]: `Este é um item privado. Ao editá-lo, as mudanças serão refletidas em todas páginas e pranchas que o item está sendo utilizado.`

File: config/locales/en.yml [hunk 2]
```diff
@@ -426,6 +426,7 @@ en:
     not_linked: -- not linked --
     og_description: Alternative Communication app that allows people with speech and mobility impairments to communicate by using pictograms that represent words.
     page_destroy_alert: Deleting this page will also delete all of its pictograms.
+    page_item_edit_alert: This is a private item, editing it will change all references.
```
[the remark this answers is not in the document]
Comment (longaraymatheus) [reply config/locales/en.yml]: `This is a private item. Once edited, changes will be reflected in all other pages and boards this item is being used.`

**Comment by leandrohmanica** [comment]
Thanks @longaraymatheus , I really apreciate yur contribuitions!

**Pull request merged**